### PR TITLE
Remove flow-typed from repo

### PR DESCRIPTION
**Pull request by pronebird**
Checklist for a PR:

* [X] Describe the change in **`CHANGELOG.md`**. Only applicable if the change has any impact for a user.

This PR removes flow-typed from repository. Travis will always install the latest annotations and will error if something is wrong in the source code, which is great as an early detection of a coming trouble.

It's up to developer to keep annotations up to date now.

Please use `yarn run flow-typed install` to install annotations and `yarn run flow-typed update` to update them once in a while.

---
This change is [Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/215)

### CODE REVIEW

**Comment by faern** [comment]
Reviewed 57 of 66 files at r1.
Review status: 0 of 66 files reviewed, 1 unresolved discussion (waiting on @jvff, @faern, and @pronebird)

---

*[.travis.yml, line 14 at r1](https://reviewable.io/reviews/mullvad/mullvadvpn-app/215#-LEuFW7G9mIedPMTUoR2:-LEuFW7H7-o92d--kprR:b-v4rulv) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/85ec297c0d5be773393b94b45e766f984eda97fe/.travis.yml#L14)):*
> ```YAML
>       install:
>         - yarn install
>         - yarn run flow-typed install
> ```

We probably have to add the same to `build.sh` to make the builds produced by that script make sure we have the latest stuff?

---


*Comments from [Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/215)*

**Comment by pronebird** [comment]
Review status: 0 of 66 files reviewed, 1 unresolved discussion (waiting on @jvff, @faern, and @pronebird)

---

*[.travis.yml, line 14 at r1](https://reviewable.io/reviews/mullvad/mullvadvpn-app/215#-LEuFW7G9mIedPMTUoR2:-LEuFwtA38btqOJjysBC:bf6nrnt) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/85ec297c0d5be773393b94b45e766f984eda97fe/.travis.yml#L14)):*
<details><summary><i>Previously, faern (Linus Färnstrand) wrote…</i></summary><blockquote>

We probably have to add the same to `build.sh` to make the builds produced by that script make sure we have the latest stuff?
</blockquote></details>

If we do not run `yarn lint` there is no need for that. This is purely for development purposes.

---


*Comments from [Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/215)*

**Comment by jvff** [comment]
Reviewed 66 of 66 files at r1.
Review status: 57 of 66 files reviewed, 1 unresolved discussion (waiting on @faern and @jvff)

---



*Comments from [Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/215)*

**Comment by pronebird** [comment]
Review status: 57 of 66 files reviewed, 1 unresolved discussion (waiting on @faern, @jvff, and @pronebird)

---

*[.travis.yml, line 14 at r1](https://reviewable.io/reviews/mullvad/mullvadvpn-app/215#-LEuFW7G9mIedPMTUoR2:-LEuG_tqD9aEneYDJS0H:b-iwm4st) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/85ec297c0d5be773393b94b45e766f984eda97fe/.travis.yml#L14)):*
<details><summary><i>Previously, pronebird (Andrei Mihailov) wrote…</i></summary><blockquote>

If we do not run `yarn lint` there is no need for that. This is purely for development purposes.
</blockquote></details>

I meant `yarn flow` not `lint`. It does not affect `lint` IIRC.

---


*Comments from [Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/215)*

**Comment by faern** [comment]
Reviewed 9 of 66 files at r1.
Review status: :shipit: complete! all files reviewed, all discussions resolved

---

*[.travis.yml, line 14 at r1](https://reviewable.io/reviews/mullvad/mullvadvpn-app/215#-LEuFW7G9mIedPMTUoR2:-LEuGXHN18e6DHNEQKD7:bi8kq97) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/85ec297c0d5be773393b94b45e766f984eda97fe/.travis.yml#L14)):*
<details><summary><i>Previously, pronebird (Andrei Mihailov) wrote…</i></summary><blockquote>

If we do not run `yarn lint` there is no need for that. This is purely for development purposes.
</blockquote></details>

Oh. So the flow-type stuff is only for linting? Well good, then it can be left out of the build process for final artifacts.

---


*Comments from [Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/215)*

**Comment by pronebird** [comment]
Review status: :shipit: complete! all files reviewed, all discussions resolved

---

*[.travis.yml, line 14 at r1](https://reviewable.io/reviews/mullvad/mullvadvpn-app/215#-LEuFW7G9mIedPMTUoR2:-LEuGgL4D7dGdcErBAPL:bkzstfd) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/85ec297c0d5be773393b94b45e766f984eda97fe/.travis.yml#L14)):*
<details><summary><i>Previously, faern (Linus Färnstrand) wrote…</i></summary><blockquote>

Oh. So the flow-type stuff is only for linting? Well good, then it can be left out of the build process for final artifacts.
</blockquote></details>

Basically yes. It's only to ensure type safety in application logic. Javascript cares not about types whatsoever... 

---


*Comments from [Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/215)*
